### DISCUSSION TITLE
How to have a route which is part static and part dynamic?

### DISCUSSION THREAD

**Discussion by yaldakarimi** [vercel/next.js#22758] [Help]
Hello guys.
The issue I am facing is how to have a route that looks sth like xyz-[id].js .
I want the xyz to appear before the id and to be separated from the id by a dash. 
is it possible to do so?

**Comment by VinayaSathyanarayana** [comment]
Can you use [rewtites](https://nextjs.org/docs/api-reference/next.config.js/rewrites)?
Also see:  https://github.com/vercel/next.js/discussions/22012 and https://github.com/vercel/next.js/discussions/22333

**Reply by VinayaSathyanarayana** [reply]
Did my suggestion work for you
